### ISSUE TITLE
unable to reproduce given validation BLEU-4 score.

### ISSUE THREAD

**Issue by mayankatsrijan** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#90]
I'm not able to reproduce the given mentioned validation BLEU-4 score in README.
after 20 epochs without finetuning, below are my scores. Don't know, what's wrong?
LOSS - 3.220, TOP-5 ACCURACY - 76.039, BLEU-4 - 0.22422202369761812

I'm using PyTorch latest version.

**Comment by mayankatsrijan** [comment]
Finally, I found that nltk calculates BLEU on a 0-1 scale but sometimes people also use 0-100 scale by multiplying the same 0-1 scale by 100. So, it thus confirms that above score 0.2242 can be expressed as 22.44 as well. It should be noted in README as well for other's convenience and to avoid confusion.

**Comment by ada-dl** [comment]
Thank you for catching this @mayankatsrijan is there a way to fix this in the code so that the score is printed as 22.44 in the example ? I am having similar results after training
